onnxruntime/core/session/inference_session.h

```diff
@@ -132,6 +139,8 @@ class InferenceSession {
   */
   common::Status LoadCustomOps(const std::vector<std::string>& dso_list);
```

Comment:
Is this API still needed?

Reply:
This is the existing shared library way to load custom ops, I didn't change it. Is anyone using it currently?

Reply:
Let's not touch this (remove it maybe) in this PR please. It may be done in separate PR if needed. Make sense please?

Reply:
No body is using this API today and relies on a different spec. We can remove it in a separate PR.